flexbox-practice.html

```diff
@@ -0,0 +1,91 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Flexbox Practice Site</title>
+    <link rel="stylesheet" href="css-flexbox-practice.css">
+</head>
+<body>
+    <div class="site">
+        <nav class="navigation">
+            <img src="http://placehold.it/" alt="logo">
+            <ul>
+                <li><a href="#">about US</a></li>
+                <li><a href="#">Products</a></li>
+                <li><a href="#">Contacts</a></li>
+            </ul>           
+        </nav>
+        <div class="section-1">
+            <div class="section-1-left">        
+                <img src="http://placehold.it/" class="img" alt="image">
+            </div>
+            <div class="section-1-right">
+                <h2>Title</h2>
+                <p>Some Text Down Here</p>
+                <button>Button</button>
+                <button class="m">Button</button>
+            </div>
+        </div>
+        <div class="section-2">
+            <div class="box">
+                <h2>Title</h2>
+                <p>Some Text</p>
+            </div>
+            <div class="box">
+                <h2>Title</h2>
+                <p>Some Text</p>
+                <button class="btn-2">Button</button>
+            </div>
+            <div class="box">
+                <h2>Title</h2>
+                <button>Button</button>
+                <p>Some Text</p>
+            </div>    
+        </div>
+        <div class="section-3">
+            <h2>Some Heading</h2>
+            <img src="http://placehold.it/" alt="small image">
+            <p>Text Here Text</p>
+            <p>Here Text Here Text</p>
+            <p>Here Text Here</p>
+        </div>
+        <footer>
+            <div>
+                <img src="http://placehold.it" alt="logo">
+            </div>
+            <div>
+                <ul class="footer-list">
```

Comment:
The use of `footer-list` and `p-links` below is what you should be doing when you want to style an HTML tag.
Do not directly style `ul` or `li` in your css, unless you are absolutely sure that every instance of `ul` and `li` will use those styles.  This applies to ALL html tags... example: <div>, <header>, <section>... etc... you would not directly style these html elements, you add classes or id's and then style those classes or id's